notebooks/logging-examples/logging-plots.ipynb

```diff
@@ -0,0 +1,330 @@
+   "metadata": {},
+   "source": [
+    "We can also create a plot and covert it to bytes to save it in memory. Using `to_image` we can convert our plot to bytes in-memory. We'll later log the plot from in-memory with `Rubicon-ml`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "8544305b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bar_plot=px.bar(df, x=\"direction\", y=\"frequency\", color=\"strength\")\n",
+    "bar_plot_bytes=bar_plot.to_image(format=\"png\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4a41ac4b",
+   "metadata": {},
+   "source": [
+    "## Logging your plot\n",
+    "In `Rubicon_ml` , `artifacts` are a catch-all for any other type of data that can be logged to a file. We can use artifacts to capture information that doesn't fall into the `experiment`, `dataframe`, `feature`, and  `parameter` category. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4bc3f6aa",
+   "metadata": {},
+   "source": [
+    "Leveraging `artifacts`, plots can be logged just by passing its file location in the `data_path` argument to `project.log_artifact()`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "d0823f93",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<rubicon_ml.client.artifact.Artifact at 0x16624a1d0>"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "project.log_artifact(data_path=\"saved-scatter-plot-for-logging.png\", description=\"bar plot logged with path\")\n",
+    "artifact_plot_from_file=project.artifacts()[0]\n",
+    "artifact_plot_from_file"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "420a7c6d",
+   "metadata": {},
+   "source": [
+    "We can also log plots by passing its byte representation to the `data_bytes` argument to `project.log_artifact()` as well."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "b064e579",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<rubicon_ml.client.artifact.Artifact at 0x166e75a80>"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "project.log_artifact(name=\"image\", data_bytes=bar_plot_bytes)\n",
+    "artifact_plot_bytes=project.artifacts()[1]\n",
+    "artifact_plot_bytes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a9eb5364",
+   "metadata": {},
+   "source": [
+    "## Retrieving your logged plot programmatically "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fbb31fb1",
+   "metadata": {},
+   "source": [
+    "Since the `PNG` representation of both plots were logged, we can retrieve our artifacted plots programmatically as well. Using `io` and `PIL` library we can convert our plots into an `PNG` image."
```

Comment:
```suggestion
    "Since the `PNG` representations of both plots were logged, we can retrieve our artifact plots programmatically. Using `io` and `PIL` we can visualize our plots as `PNG` images."
```